integrations.json

```diff
@@ -105,5 +105,58 @@
         }
       }
     ]
+  },
```

Comment:
I do not found any script to refresh this file in case of changes in binary instrumentation. Do we have anything like this?